.env.example

```diff
@@ -5,9 +5,23 @@ GITHUB_ANALYZER_PASSWORD=postgres
 # Must be set as the webhook secret on your github app
 GITHUB_ANALYZER_WEBHOOK_SECRET=fake-secret
 
+# Own Github organization
+GITHUB_ORGANIZATION=rootstrap
+
+# Worpress data
+WORDPRESS_SITE_ID=
+WORDPRESS_CLIENT_ID=
+WORDPRESS_CLIENT_SECRET=
+WORDPRESS_USERNAME=
+WORDPRESS_PASSWORD=
```

Comment:
Are these variables related to this particular PR?

Reply:
No, they are not! I think I originally added these for when someone new would join the team, so that they could have the correct env vars. But well, I finished a bit too late 😅 
In any case, I could totally add these changes in another PR

Reply:
Personally I think it's fine